core/src/test/scala/com/stripe/dagon/DataFlowTest.scala

```diff
@@ -833,7 +832,8 @@ class DataFlowTest extends FunSuite {
       if (incs <= 0) f
       else incrementChain(f.map(_ + 1), incs - 1)
 
-    val incCount = if (catalysts.Platform.isJvm) 10000 else 1000
+    //val incCount = if (catalysts.Platform.isJvm) 10000 else 1000
```

Comment:
I think testing this with 1000 destroys the test. This is about the stack, I think... but we can deal with it later.

Reply:
Yeah, catalysts isn't published for 2.13 so I was gonna disable this then try to fix it.